scripts/exgfs_atmos_grib2_special_npoess.sh

```diff
@@ -40,13 +40,49 @@ export opt26=' -set_grib_max_bits 25 -fi -if '
 export opt27=":(APCP|ACPCP|PRATE|CPRAT|DZDT):"
 export opt28=' -new_grid_interpolation budget -fi '
 
-SLEEP_LOOP_MAX=$(expr $SLEEP_TIME / $SLEEP_INT)
+####################################
+# Specify Timeout Behavior of Post
+#
+# SLEEP_TIME - Amount of time to wait for
+#              a restart file before exiting
+# SLEEP_INT  - Amount of time to wait between
+#              checking for restart files
+####################################
+export SLEEP_TIME=${SLEEP_TIME:-900}
+export SLEEP_INT=${SLEEP_TIME:-5}
+
+SLEEP_LOOP_MAX=$(expr ${SLEEP_TIME} / ${SLEEP_INT})
+
+# TODO: Does this section do anything? I retained if for clarity of
+# changes/updates, but it does not appear to do anything.
```

Comment:
I traced this history all the way back to when it was created. It has always been a do-nothing block of code. Seems like there should be an exit in here if it has already been generated. But, something for another PR. Keep the TODO.